src/telegram.js

```diff
@@ -845,6 +845,165 @@ class TelegramBot extends EventEmitter {
   }
 
   /**
+   * Use this method to restrict a user in a supergroup.
+   * The bot must be an administrator in the supergroup for this to work
+   * and must have the appropriate admin rights. Pass True for all boolean parameters
+   * to lift restrictions from a user. Returns True on success.
+   *
+   * @param {Number|String} chatId Unique identifier for the target chat or username of the target supergroup
+   * @param {String} userId
+   * @param {Object} [options] Additional Telegram query options
+   * @returns {Promise}
+   * @see https://core.telegram.org/bots/api#restrictchatmember
+   */
+  restrictChatMember(chatId, userId, form = {}) {
+    form.chat_id = chatId;
+    form.user_id = userId;
+    return this._request('restrictChatMember', { form });
+  }
+
+  /**
+   * Use this method to promote or demote a user in a supergroup or a channel.
+   * The bot must be an administrator in the chat for this to work
+   * and must have the appropriate admin rights. Pass False for all boolean parameters to demote a user.
+   * Returns True on success.
+   *
+   * @param {Number|String} chatId Unique identifier for the target chat or username of the target supergroup
+   * @param {String} userId
+   * @param {Object} [options] Additional Telegram query options
+   * @returns {Promise}
+   * @see https://core.telegram.org/bots/api#promotechatmember
+   */
+  promoteChatMember(chatId, userId, form = {}) {
+    form.chat_id = chatId;
+    form.user_id = userId;
+    return this._request('promoteChatMember', { form });
+  }
+
+  /**
+   * Use this method to export an invite link to a supergroup or a channel.
+   * The bot must be an administrator in the chat for this to work and must have the appropriate admin rights.
+   * Returns exported invite link as String on success.
+   *
+   * @param {Number|String} chatId Unique identifier for the target chat or username of the target supergroup
+   * @param {Object} [options] Additional Telegram query options
```

Comment:
No additional parameters for this method see [here](https://core.telegram.org/bots/api#exportchatinvitelink).